src/pages/Defi/views/StakingVaults.tsx

```diff
@@ -12,9 +16,48 @@ const DefiHeader = () => {
   )
 }
 
+const FoxFarmCTA = () => {
+  const ethLogoSrc = 'https://assets.coincap.io/assets/icons/256/eth.png'
+  const foxLogoSrc = 'https://assets.coincap.io/assets/icons/256/fox.png'
+  return (
+    <Card variant='outline' my={1}>
+      <Stack
+        direction={{ base: 'column', xl: 'row' }}
+        alignItems='center'
+        justifyContent='space-between'
+        px={10}
+        py={4}
+      >
+        <Flex alignItems='center'>
+          <RawText alignSelf='end' fontSize={30} mr={-2} mb={{ base: '2', xl: '-2' }} zIndex={1}>
+            🚜
+          </RawText>
+          <AssetIcon mr={-2} boxSize='40px' src={ethLogoSrc} />
+          <AssetIcon boxSize='40px' src={foxLogoSrc} />
```

Comment:
```suggestion
          <Flex alignItems='flex-end'>
            <RawText lineHeight={'1.1'} fontSize={30} zIndex={1}>
              🚜
            </RawText>
            <AssetIcon ml={-3} boxSize='40px' src={ethLogoSrc} />
            <AssetIcon ml={-2} boxSize='40px' src={foxLogoSrc} />
          </Flex>
```

- Don't play with margins in order to align the first letter
- If you set his lineHeight lower as I submited, you will be able to align it using flexbox
- As a good practice, never apply negatives margin rights, as if the latest element is removed, it will cause some bugs